Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:2.7-alpine3.7
+
+RUN apk update && \
+    apk add openssh-client git unzip vim shadow
+
+RUN useradd -ms /bin/bash user
+
+WORKDIR /home/user
+
+ADD . /home/user/env2consul
+
+WORKDIR env2consul
+
+RUN pip install -r requirements.txt && \
+    chown user:user /home/user -R
+
+USER user
+
+RUN mkdir $HOME/.env2consul && \
+    mkdir $HOME/.env2consul/devops-env2consul && \
```

Comment:
This directory wont be relevant to anyone else - but they will need their own repo name here. 

Seems more like a bug in the python script, probably should fix it there (ensure it creates the directory before the first repo clone)